vertx-web/src/main/java/io/vertx/ext/web/handler/SessionHandler.java

```diff
@@ -123,4 +129,12 @@ static SessionHandler create(SessionStore sessionStore) {
   @Fluent
   SessionHandler setSessionCookieName(String sessionCookieName);
 
+  /**
+   * Set the session cookie name
+   *
+   * @param minLength  the session id minimal length
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  SessionHandler setMinLength(int minLength);
```

Comment:
The comments for this method are incorrect. It also looks like the implementations do not respect this minimum when generating session ids.